src/stream_parser/file_reader.rs

```diff
@@ -220,8 +217,27 @@ impl<'c> LogParser<'c> {
                         "flag bits at bad position",
                     ));
                 }
+                let flag_bits = parse_flag_bits(&msg)?;
+
+                // Check for incompatible flag bits. If there's any unknown bits set, we cannot
+                // parse the log
+                let ulog_incompat_flag0_data_appended_mask = 1u8 << 0;
+                if (flag_bits.incompat_flags[0] & !(ulog_incompat_flag0_data_appended_mask)) != 0 {
```

Comment:
Is `ulog_incompat_flag0_data_appended_mask` not a constant at this point?

Reply:
Is there a specific convention for constants?

Reply:
Rust constants are basically the same as C (https://doc.rust-lang.org/rust-by-example/custom_types/constants.html). 

But I misread this line. I didn't initially see that this was doing bitwise logic. I read the `&` as an AND (`&&`). There's no need to make it a constant.